migrator/src/main/java/uk/nhs/digital/ps/migrator/task/importables/NhsOutcomesFrameworkImportables.java

```diff
@@ -0,0 +1,62 @@
+package uk.nhs.digital.ps.migrator.task.importables;
+
+import uk.nhs.digital.ps.migrator.model.hippo.Folder;
+import uk.nhs.digital.ps.migrator.model.hippo.HippoImportableItem;
+import uk.nhs.digital.ps.migrator.model.hippo.Publication;
+import uk.nhs.digital.ps.migrator.model.nesstar.Catalog;
+import uk.nhs.digital.ps.migrator.model.nesstar.CatalogStructure;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Stream;
+
+import static java.util.stream.Collectors.toList;
+import static uk.nhs.digital.ps.migrator.task.ImportableItemsFactory.*;
+
+public class NhsOutcomesFrameworkImportables {
+
+    static public List<HippoImportableItem> create(final CatalogStructure catalogStructure, final Folder ciRootFolder) {
+
+        // Target CMS structure:
+        //
+        // A)  NHS Outcomes Framework                               FOLDER
+        // B)  Outcomes Framework publication 'content' document    PUBLICATION
+        // C)    folders per node                                   FOLDER
+        // D)      DataSet documents                                DATASET
+
+        // A)
+        final Catalog rootCatalog = catalogStructure.findCatalogByLabel("NHS Outcomes Framework");
+        final Folder rootFolder = toFolder(rootCatalog, ciRootFolder);
+
+        // B)
+        final Publication publication = toPublication(rootCatalog, rootFolder);
+
+        // C & D)
+        final List<HippoImportableItem> domainsWithDatasets = getImportablesFromCatalog(rootCatalog, rootFolder)
+            .collect(toList());
+
+        final List<HippoImportableItem> importableItems = new ArrayList<>();
+        importableItems.add(rootFolder);
+        importableItems.add(publication);
+        importableItems.addAll(domainsWithDatasets);
+
+        return importableItems;
+    }
+
+    private static Stream<HippoImportableItem> getImportablesFromCatalog(Catalog rootCatalog, Folder rootFolder) {
+        return rootCatalog.getChildCatalogs().stream()
```

Comment:
My apologies for having failed to word this requirement clearly, but node '_NHS Outcomes Framework (NHS OF) summary dashboard and useful links_' needs to be excluded from the import.

Adding something like the following after the `.stream()` and before the `.flatMap(...)` calls should do the trick:
```
.filter(catalog -> !"NHS Outcomes Framework (NHS OF) summary dashboard and useful links".equals(catalog.getLabel()))
```

Reply:
I'm on it